### ISSUE TITLE
Is this project actively being maintained?

### ISSUE THREAD

**Issue by bmiller59** [wallix/PEPS#79]
Hello. I am excited about using and contributing to PEPS but after reviewing the commit history (very little in the past year) I am not sure if this repo is being actively maintained or not. Can someone clarify and indicate if there is a more active fork that has replaced this repo? Thank you.

**Comment by hbbio** [comment]
The README has been updated: This version and repository is not maintained anymore. But we will be releasing a much enhanced product in June 2017.

**Comment by bmiller59** [comment]
Thank you for updating the README. Is the roadmap (e.g. anticipated features) accurate in the README so we be aware of what is going to be added?

I also ask because I am seeking a project like this one to start contributing to. 

Thanks for such an interesting project. Looking forward to the new version.

**Comment by hbbio** [comment]
Yes the roadmap is still the same but the technical architecture has been moved to micro services.

**Comment by roshanjonah** [comment]
Hi there, Would you say it would be easy to upgrade to the new version once it is released or will you not support an upgrade from this version?

**Comment by biernacki** [comment]
@hbbio Any news regarding the new version? The anticipation is killing me ;)

**Comment by asifsaud** [comment]
Will the new product be licensed as GPL or will it be commercial? just want to know if I should wait or move on :)
